docs/api-docs/cart-and-checkout/cart-and-checkout-overview.mdx

```diff
@@ -98,7 +98,7 @@ fetch(`/api/storefront/order/${jsContextOrderId}`, {
 ## Server-to-Server Cart and Checkout
```

Comment:
change all instances of `Server-to-Server` to `REST Management`, then explain that they are backend APIs as opposed to the frontend Storefront APIs.